### ISSUE TITLE
N64 achievements are up.

### ISSUE THREAD

**Issue by grenade44** [meleu/RetroArch-problematic-cheevos#21]
Hi,

There are 2 set of n64 achievements which are up. http://retroachievements.org/gameList.php?c=2

Haven't had a chance to test with any emulators yet.

**Comment by meleu** [comment]
do you know what is the "official unofficial emulator" for N64?

**Comment by grenade44** [comment]
Its project 64,

**Comment by grenade44** [comment]
I'm a developer on RA and have access to the beta.

**Comment by grenade44** [comment]
There is now 9 sets available, just wonder if there's any updates with bringing n64 achievements to retroarch?

**Comment by meleu** [comment]
I don't know what is the ~~best~~ recommended core to play N64 on RetroArch...

**Comment by grenade44** [comment]
I've tried one game on lr-Mupen64plus and doesn't seem to bad. Will have to test some more.

**Comment by grenade44** [comment]
@leiradel

**Comment by grenade44** [comment]
@leiradel
Just wondering if you have managed to get retroachievements working with N64 yet?

**Comment by leiradel** [comment]
Not yet :(

I'll take a look at it this week

**Comment by grenade44** [comment]
@leiradel was going to close this but I have got a couple of issues.

I updated retroarch and lr-mupen64plus. Achievements dont load in Golden eye or paper Mario but the list does load in Super Mario 64 but doesnt show any achievements as being unlocked. I had unlocked 1 via retroachievements offical emulator.

**Comment by leiradel** [comment]
The games hashes have to be updated at Retro Achievements, the official emulator uses an internally computed hash and RetroArch uses the plain ROM hash. This will be incrementally solved as the new hashes are updated.

As for unlocked cheevos, the logic is exactly the same as for other platforms and I do see my unlocked N64 cheevos in RetroArch when it loads. Can you double check please?

**Comment by grenade44** [comment]
Goldeneye
[INFO] CHEEVOS got game id 0.
[INFO] CHEEVOS getting game id for hash 2a56a06a05a9dbf9e693325b503ab5b1
[INFO] CHEEVOS this game doesn't feature achievements.

Paper Mario
[INFO] CHEEVOS getting game id for hash 028c27ebef2cbe476236171cd294b0b0
[INFO] CHEEVOS got game id 0.
[INFO] CHEEVOS this game doesn't feature achievements.

Super Mario 64
[INFO] CHEEVOS got game id 10003.
[INFO] CHEEVOS deactivated unlocked cheevo 48638 (A new journey).
[INFO] CHEEVOS deactivated unlocked achievements in softcore mode.
[INFO] CHEEVOS deactivated unlocked achievements in hardcore mode.

Blast Corps
[INFO] CHEEVOS getting game id for hash e579aa236d12725597dd4ed8c3e9350b
[INFO] CHEEVOS got game id 0.
[INFO] CHEEVOS this game doesn't feature achievements.

**Comment by grenade44** [comment]
@leiradel Is there a way for me to help update the hashes?

**Comment by leiradel** [comment]
So the unlocked cheevos are showing like so in the UI right?

I'm not sure how to update the hashes, maybe @GameDragon2k can give you a hand here.

**Comment by grenade44** [comment]
No its showing as locked

**Comment by leiradel** [comment]
**A new journey** is showing as locked? It shows as unlocked for me:
![is1](https://user-images.githubusercontent.com/357536/27997987-97dee050-64fb-11e7-92fb-7bf09b3ce18d.png)
![is2](https://user-images.githubusercontent.com/357536/27997989-9b848002-64fb-11e7-8673-91b8fe0f7d18.png)
Can you post a verbose log with `|grep CHEEVOS`? Thanks.

**Comment by grenade44** [comment]
@leiradel Ive worked out the issue, when I played the offical emulator I didnt have hardcore on but I do have it turned on my Retorpie/RetroArch machine. so when it loaded it was reading the hardcore achievement number and I was checking the hardcore achievement list. It works fine.

Also the hashes seemed to have been updated. goldeneye now works.

Going to close, Will open a new issue if i find anymore problems.
